### PR TITLE
[Python] Client API HowTo articles batch [Replace C# samples]

### PR DESCRIPTION
**Pull request by reebhub**
Related issues: 
[RDoc-2750](https://issues.hibernatingrhinos.com/issue/RDoc-2750/Python-Client-API-Session-How-to-Clear-a-session-Replace-C-samples) - Clear a session
[RDoc-2751](https://issues.hibernatingrhinos.com/issue/RDoc-2751/Python-Client-API-Session-How-to-Evict-entity-from-a-session-Replace-C-samples) - Evict entity from a session
[RDoc-2752](https://issues.hibernatingrhinos.com/issue/RDoc-2752/Python-Client-API-Session-How-to-Refresh-entity-Replace-C-samples) - Refresh entity
[RDoc-2753](https://issues.hibernatingrhinos.com/issue/RDoc-2753/Python-Client-API-Session-How-to-defer-commands-Replace-C-samples) - defer commands
[RDoc-2754](https://issues.hibernatingrhinos.com/issue/RDoc-2754/Python-Client-API-Session-How-to-Perform-requests-lazily-Replace-C-samples) - Perform requests lazily
[RDoc-2755](https://issues.hibernatingrhinos.com/issue/RDoc-2755/Python-Client-API-Session-How-to-Get-entity-change-vector-Replace-C-samples) - Get entity change vector
[RDoc-2756](https://issues.hibernatingrhinos.com/issue/RDoc-2756/Python-Client-API-Session-How-to-Get-entity-last-modified-Replace-C-samples) - Get entity last modified
[RDoc-2757](https://issues.hibernatingrhinos.com/issue/RDoc-2757/Python-Client-API-Session-How-to-Get-entity-counters-Replace-C-samples) - Get entity counters

### CODE REVIEW

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-change-vector.python.markdown [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+{CODE:python get_change_vector_1@ClientApi\Session\HowTo\GetChangeVector.py /}
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| **entity** | `object` | Instance of an entity for which an Etag will be returned |
+
+| Return Type | Description |
+| ----------- | ----------- |
+| `str` | Returns the current change-vector for an entity. Throws an exception if the `instance` is not tracked by the session. |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-change-vector.python.markdown:20]:
```suggestion
| `str` | Returns the current change-vector for an entity. Throws an exception if the `entity` is not tracked by the session. |

```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-counters.python.markdown [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+{CODE:python syntax@ClientApi\Session\HowTo\GetCountersFor.py /}
+
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| **entity** | `object` | Instance of an entity for which counter names will be returned. |
+
+| Return Type | Description |
+| ----------- | ----------- |
+| `List[str]` | Returns the counter names for the specified entity, or `None` if the entity has no counters.<br>An exception is thrown if the `object` is not tracked by the session. |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-counters.python.markdown:39]:
```suggestion
| `List[str]` | Returns the counter names for the specified entity, or `None` if the entity has no counters.<br>An exception is thrown if the `entity` is not tracked by the session. |
```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-last-modified.python.markdown [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+{CODE:python get_last_modified_1@ClientApi\Session\HowTo\GetLastModified.py /}
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| **entity** | `object` | Instance of an entity for which the last modified date will be returned. |
+
+| Return Type | Description |
+| ----------- | ----------- |
+| `datetime` | Returns the last modified date for an entity. Throws an exception if the `object` is not tracked by the session. |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-last-modified.python.markdown:19]:
```suggestion
| `datetime` | Returns the last modified date for an entity. Throws an exception if the `entity` is not tracked by the session. |

```

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/refresh-entity.python.markdown [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+To update an entity with the latest changes from the server, use the `refresh` method from `advanced` session operations.
+
+## Syntax
+
+{CODE:python refresh_1@ClientApi\Session\HowTo\Refresh.py /}
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| **entity** | `object` | Instance of an entity that will be refreshed |
```
Comment (poissoncorp) [inline Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/refresh-entity.python.markdown:11]:
```suggestion
| **entity** | `object` | An entity that will be refreshed |

```

**Review by poissoncorp — APPROVED** [review]

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-change-vector.python.markdown (hunk 1 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-change-vector.python.markdown:20]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-counters.python.markdown (hunk 2 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-counters.python.markdown:39]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-last-modified.python.markdown (hunk 3 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/get-entity-last-modified.python.markdown:19]: done

File: Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/refresh-entity.python.markdown (hunk 4 above)
Comment (reebhub) [reply Documentation/5.4/Raven.Documentation.Pages/client-api/session/how-to/refresh-entity.python.markdown:11]: done

**Pull request merged**